main.go

```diff
@@ -221,6 +279,8 @@ func main() {
 	outputFileName := flag.String("outputFileName", "", "outputFile to encrypt/decrypt")
 	shouldEncrypt := flag.Bool("encrypt", false, "Should the application encrypt the message")
 	shouldDecrypt := flag.Bool("decrypt", false, "Should the application decrypt the ciphertext")
+	shouldSign := flag.Bool("signOnly", false, "Should the application sign the message")
+	shouldVerify := flag.Bool("verifyOnly", false, "Should the application verify the input blob")
```

Comment:
why not just `sign` and `verify`?

Reply:
Good point.. it's definitely easier and makes more sense. I use signOnly to make it consistent with SDK